app/services/orders/state_service.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Orders
+  class StateService
+    include AASM
+    attr_reader :order
+
+    def initialize(order)
+      @order = order
```

Comment:
Did you need it here? it hasn't been use

Reply:
I do need it for for:
```
    def payment_completed?
      Payments::StateService.new(order.payment).completed?
    end
```

```
    def shipment_shipped?
      Shipments::StateService.new(order.shipment).shipped?
    end
```

similar case for Shipments::StateSeervice, but I can get rid of attr_reader in payments, and applied this change